.github/workflows/nightly-gauntlet.yaml

```
@@ -143,7 +143,7 @@ jobs:
 
           DB=ci gotestsum \
             --format standard-quiet --packages "./..." \
-            -- -timeout=20m -v -p $NUM_PARALLEL_PACKAGES -parallel=$NUM_PARALLEL_TESTS $TESTCOUNT
+            -- -timeout=20m -v -p "$NUM_PARALLEL_PACKAGES" -parallel="$NUM_PARALLEL_TESTS $TESTCOUNT"
```

shouldn't NUM_PARALLEL_TESTS and TESTCOUNT be quotes separately?
```suggestion
            -- -timeout=20m -v -p "$NUM_PARALLEL_PACKAGES" -parallel="$NUM_PARALLEL_TESTS" "$TESTCOUNT"
```

---

yeah, I fat fingered that one 😅